android/app/src/main/java/com/zulipmobile/MainActivity.kt

```diff
@@ -43,6 +43,11 @@ open class MainActivity : ReactActivity() {
     }
 
     private fun handleSend(intent: Intent) {
+        // Intent is reused after quitting, skip it.
+        if((getIntent().flags and Intent.FLAG_ACTIVITY_LAUNCHED_FROM_HISTORY)!=0){
+            return;
+        }
```

Comment:
Huh, that is *wacky* that it behaves that way! Good find.

Reply:
Took me several hours to figure this out, I tried a lot of different things 😰